### ISSUE TITLE
Data Files for NQ

### ISSUE THREAD

**Issue by Gouzi3618** [ArvinZhuang/DSI-QG#4]
Hi @ArvinZhuang,

Thanks for your great efforts! I was wondering if it is possible to provide the data files for the NQ dataset? Thanks!

**Comment by ArvinZhuang** [comment]
Hi, thanks for your kind words :)

You can take a look at our old [repo](https://github.com/ArvinZhuang/DSI-transformers), in which we have code for processing NQ dataset.

**Comment by Gouzi3618** [comment]
Get it, thanks! I was wondering if the results are improved with the current DSI-QG code?

**Comment by ArvinZhuang** [comment]
yeah, I think so. I remember scores can be improved by just randomly assigning docids rather than assigning ids sequentially.
